cw/capture.py

```diff
@@ -1193,21 +1263,99 @@ def capture_ecdsa_stream(ot, fw_bin, pll_frequency, capture_cfg):
     # Currently, we need to reprogram the firmware before every sign operation
     # TODO: Investigate the C code (ecc384_serial) to check if we can run multiple
     #       signing operations without reloading the firmware.
-    reset_firmware = True
+    reset_firmware = False
 
     project = cw.create_project(capture_cfg["project_name"], overwrite=True)
 
     # Enable the streaming mode
     if ot.scope._is_husky:
         ot.scope.adc.stream_mode = True
-        ot.scope.adc.bits_per_sample = 8
+        ot.scope.adc.bits_per_sample = 12
```

Comment:
Could you add a comment  here how to determine the appropriate values?
It might be obvious now, but later one might wonder where this magic number comes from.

Reply:
Good point :) I added a comment and included a link to NewAE reference document regarding the CW-Husky streaming mode.